src/lib/containers/download_list_v2/RequestDownloadCard.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react'
+import { useErrorHandler } from 'react-error-boundary'
+import { toError } from '../../utils/ErrorUtils'
+import { SynapseSpinner } from '../LoadingScreen'
+import { useGetEntityHeaders } from '../../utils/hooks/SynapseAPI/useGetEntityHeaders'
+import { EntityHeader } from '../../utils/synapseTypes'
+import { Icon } from '../row_renderers/utils'
+import { VARIABLE_DIFFICULTY } from '../../utils/SynapseConstants'
+import { Button } from 'react-bootstrap'
+
+
+export type RequestDownloadCardProps = {
+  entityId: string
+  count: number
+}
+
+export const REQUEST_DOWNLOAD_TITLE = 'Requires Download Permission'
+export const RequestDownloadCard:React.FunctionComponent<RequestDownloadCardProps> = (
+  {
+    entityId,
+    count,
+  }: RequestDownloadCardProps
+) => {
+  const handleError = useErrorHandler()
+  const {
+    data,
+    isFetching,
+    isError,
+    error: newError,
+  } = useGetEntityHeaders([{targetId: entityId}])
+  const entityHeader:EntityHeader | undefined = data?.results[0]
+  useEffect(() => {
+    if (isError && newError) {
+      handleError(toError(newError))
+    }
+  }, [isError, newError, handleError])
+
+  return (
+    <>
+      {!isError && !isFetching && (
+        <div className="RequestDownloadCart actionRequiredCard">
+          <Icon type={VARIABLE_DIFFICULTY} />
+          <div className="metadata">
+            <div className="title">{REQUEST_DOWNLOAD_TITLE}</div>
+            <div className="fileCount">{count} File(s)</div>
+            <div className="description">
+              You must be granted the download permission on <strong>{entityHeader?.name}</strong> in order to download this set of files.
+            </div>
+          </div>
+          <a className="startButtonContainer" href={`https://www.synapse.org/#!Synapse:${entityHeader?.id}`} target='_blank' rel='noreferrer'>
```

Comment:
This entity points to the "benefactor" for the file(s) that the user wants to download, which is typically a Folder or Project.  So this link will open up a new window and show the container that has the Sharing Settings that need to be modified (to grant the current user the DOWNLOAD access type)

Reply:
Kind of weird UX since we don't yet have any way to submit a request...I guess that's probably coming later?

Reply:
That's the hope.  It's been a long-time issue, and does not have an easy answer (because of the way responsibilities are delegated on the curation side).  We may need a way to specify a principal as a data request handler?